kafka/modis_kernels_kf.py

```diff
@@ -229,21 +230,22 @@ def get_mcd43_prior(mcd43_fstring, band):
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b06.vrt")),
                 gdal.Open(os.path.join(the_dir, "brdf_2016_b07.vrt")))
 
-    output_plethora = create_output_file(g, the_dir, "test")
+    output_plethora = create_output_file(g, the_dir, "band{}".format(band))
     
-    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [] )
+    kf = MODISKernelLinearKalman(modis_obs, days, output_plethora, [], the_band=band )
```

Comment:
Are you sure? I thought this was the reason the obs were pulling the solution to something that looked like visible rather than NIR. The relevent line is:
`rho_pntr = self.observations[5 + self.the_band]`
but I'm not so familiar with observations - is 5 or 6 the first band?